packages/communication-react/review/communication-react.api.md

```diff
@@ -850,6 +851,7 @@ export const COMPOSITE_ONLY_ICONS: {
     LocalDeviceSettingsSpeaker: JSX.Element;
     LocalPreviewPlaceholder: JSX.Element;
     LocalCameraSwitch: JSX.Element;
+    notificationCircle: JSX.Element;
```

Comment:
Capital Case
`notificationCircle` is very generic.
I recommend something like `ChatButtonNotificationBubble`